src/autocomplete/databases/yql/grammar/YQL.g4

```diff
@@ -333,25 +337,25 @@ expr_list
     ;
 
 pure_column_list
-    : LPAREN an_id (COMMA an_id)* RPAREN
+    : LPAREN an_existed_column_id (COMMA an_existed_column_id)* RPAREN
```

Comment:
`existed` is not grammatically correct here, `existing` is the word you're looking for